Habitica/src/com/magicmicky/habitrpgwrapper/lib/models/tasks/Task.java

```diff
@@ -112,8 +119,8 @@ public void setId(String id) {
     /**
      * @return the notes
      */
-    public String getNotes() {
-        return notes;
+    public CharSequence getNotes() {
+        return markdownParser.parseMarkdown(notes);
```

Comment:
Done in [0092186](https://github.com/HabitRPG/habitrpg-android/commit/0092186caa145be70009227cd5d22bbb936913b7) :smile: